OrbitGl/App.cpp

```diff
@@ -965,3 +961,6 @@ void OrbitApp::CrashOrbitService(
         [crash_type, this] { crash_manager_->CrashOrbitService(crash_type); });
   }
 }
+bool OrbitApp::IsCapturing() {
+  return capture_client_->state() != CaptureClient::State::kStopped;
```

Comment:
I think this method could be moved inside CaptureClient so it can be called directly instead, so it can be used in multiple clients

Reply:
done.